tower_cli/resources/job_template.py

```diff
@@ -27,6 +27,8 @@ class Resource(models.SurveyResource):
     """A resource for job templates."""
     cli_help = 'Manage job templates.'
     endpoint = '/job_templates/'
+    dependencies = ['inventory', 'credential', 'project', 'vault_credential']
```

Comment:
the way I read "dependencies" is "these need to first be copied before this resource can be copied". Is that a fair statement?

Reply:
In this case its not the order but the the assets this asset type depends on. These relate to fields in the object that you have to resolve from an ID number to a name.

i.e. when you export a job_template you have to resolve the fields [inventory,credential,project,vault_credential] from their ID numbers to their names.

The order for import is stored in SEND_ORDER in transer/common.py